docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -1,58 +1,226 @@
-   more querying capabilities in the future.
-
 This page assumes you already know an API Root URL
 for a BigchainDB node or reverse proxy.
-It should be something like ``http://apihosting4u.net:9984``
-or ``http://12.34.56.78:9984``.
+It should be something like ``https://example.com:9984``
+or ``https://12.34.56.78:9984``.
 
 If you set up a BigchainDB node or reverse proxy yourself,
 and you're not sure what the API Root URL is,
 then see the last section of this page for help.
 
 
-API Root URL
-------------
+BigchainDB Root URL
+-------------------
 
-If you send an HTTP GET request to the API Root URL
-e.g. ``http://localhost:9984`` 
-or ``http://apihosting4u.net:9984``
-(with no ``/api/v1/`` on the end), 
-then you should get an HTTP response 
+If you send an HTTP GET request to the BigchainDB Root URL
+e.g. ``http://localhost:9984``
+or ``https://example.com:9984``
+(with no ``/api/v1/`` on the end),
+then you should get an HTTP response
 with something like the following in the body:
 
 .. code-block:: json
 
     {
+      "_links": {
+        "docs": "https://docs.bigchaindb.com/projects/server/en/v0.9.0/",
+        "api_v1": "http://example.com:9984/api/v1/"
+      },
       "keyring": [
         "6qHyZew94NMmUTYyHnkZsB8cxJYuRNEiEpXHe1ih9QX3",
         "AdDuyrTyjrDt935YnFu4VBCVDhHtY2Y6rcy7x2TFeiRi"
       ],
       "public_key": "AiygKSRhZWTxxYT4AfgKoTG4TZAoPsWoEt6C6bLq4jJR",
       "software": "BigchainDB",
-      "version": "0.6.0"
+      "version": "0.9.0"
     }
 
 
-POST /transactions/
+API Root Endpoint
+-------------------
+
+If you send an HTTP GET request to the API Root Endpoint
+e.g. ``http://localhost:9984/api/v1/``
+or ``https://example.com:9984/api/v1/``,
+then you should get an HTTP response
+that allows you to discover the BigchainDB API endpoints:
+
+.. code-block:: json
+
+    {
+      "_links": {
+        "docs": "https://docs.bigchaindb.com/projects/server/en/v0.9.0/drivers-clients/http-client-server-api.html",
+        "self": "https://example.com:9984/api/v1",
+        "statuses": "https://example.com:9984/api/v1/statuses",
+        "transactions": "https://example.com:9984/api/v1/transactions",
+      },
+      "version" : "0.9.0"
+    }
+
+Transactions
 -------------------
 
-.. http:post:: /transactions/
+.. http:get:: /transactions/{tx_id}
+
+   Get the transaction with the ID ``tx_id``.
+
+   This endpoint returns only a transaction from the ``BACKLOG`` or a ``VALID`` or ``UNDECIDED``
+   block on ``bigchain``, if exists.
+
+   :param tx_id: transaction ID
+   :type tx_id: hex string
+
+   **Example request**:
+
+   .. literalinclude:: samples/get-tx-id-request.http
+      :language: http
+
+   **Example response**:
+
+   .. literalinclude:: samples/get-tx-id-response.http
+      :language: http
+
+   :resheader Content-Type: ``application/json``
+
+   :statuscode 200: A transaction with that ID was found.
+   :statuscode 404: A transaction with that ID was not found.
```

Comment:
Following #1038, we need to add a query string `'status'`, such that by default only VALID transactions are queried. If a consumer wishes to query transactions that may be in the BACKLOG, or in an UNDECIDED block, a `'status'` query parameter must be provided, e.g.:

```text
GET /transaction/{id}?status=backlog
GET /transaction/{id}?status=undecided
GET /transaction/{id}?status=valid
```

the last case being equivalent to the default, when no status is provided.

@diminator @libscott @TimDaub could you please confirm?